README.md

```diff
@@ -28,6 +44,195 @@ The Cosmos DB Data Migration Tool core project is a C# command-line executable.
+
+    ```json
+    {
+        "SourceSettings": {            
+            "FilePath": "C:\\btcdata\\simple_json.json"
+        },
+        "SinkSettings": {
+            "ConnectionString": "AccountEndpoint=https://localhost:8081/;AccountKey=C2y6yDj...",
+            "Database": "datamigration",
+            "Container": "btcdata",
+            "PartitionKeyPath": "/id",
+            "RecreateContainer": false,
+            "IncludeMetadataFields": false
+        }
+    }
+    ```
+
+    ![The Cosmos DB Emulator Quickstart screen displays with the Primary Connection String value highlighted.](media/cosmosemulator_primaryconnectionstring.png "Cosmos DB Emulator Primary Connection String")
+
+3. Ensure the **Cosmos.DataTransfer.Core** project is set as the startup project then press <kbd>F5</kbd> to run the application.
+
+4. The application will load the provided extensions, then prompt for a **Source**. Select **JSON**. When prompted to load settings from a file enter `n` and press <kbd>Enter</kbd>. The application provides the ability to configure extensions in a separate file versus having them located in the **migrationsettings.json** file. When a separate file is not specified, the settings are read from **migrationsettings.json**. When prompted to provide the Configuration section name, press <kbd>Enter</kbd> to accept the default: **JSONSourceSettings**.
+
+    ![The Data Migration application displays with a menu prompt for Source.](media/app_source_prompt.png "Select Source")
+
+    >**Note**: Alternatively, to avoid this prompt add the **Source** property to the configuration with the value **Json** (matching the **DisplayName** property of the extension)
+
+5. The application will then prompt for a **Sink**, choose **Cosmos-nosql**. When prompted to load settings from a file enter `n` and press <kbd>Enter</kbd>. The application provides the ability to configure extensions in a separate file versus having them located in the **migrationsettings.json** file. When a separate file is not specified, the settings are read from **migrationsettings.json**. When prompted to provide the Configuration section name, press <kbd>Enter</kbd> to accept the default: **Cosmos-NoSqlSinkSettings**.
+
+    ![The Data migration application displays with a menu prompt for Sink.](media/app_sink_prompt.png "Select Sink")
+
+    >**Note**: Alternatively, to avoid this prompt add the **Sink** property to the configuration with the value **Cosmos-nosql** (matching the **DisplayName** property of the extension)
+
+6. The application then performs the data migration. After a few moments the process will indicate **Done.**.
```

Comment:
Message changed to "Data transfer complete" or "Data transfer failed"

Reply:
Updated